Mathlib/Data/List/Iterate.lean

```diff
@@ -34,7 +34,7 @@ theorem getElem?_iterate (f : α → α) (a : α) :
 @[simp]
 theorem getElem_iterate (f : α → α) (a : α) (n : ℕ) (i : Nat) (h : i < (iterate f a n).length) :
     (iterate f a n)[i] = f^[i] a :=
-  getElem_eq_iff.2 <| getElem?_iterate _ _ _ _ <| by rwa [length_iterate] at h
+  (getElem_eq_iff _).2 <| getElem?_iterate _ _ _ _ <| by rwa [length_iterate] at h
```

Comment:
Not sure what argument became explicit but all arguments to `getElem_eq_iff` should be implicit since it's an iff

Reply:
A propositional argument became explicit here: https://github.com/leanprover/lean4/pull/7672/files#diff-e45bf6797a2f201791babf1bebde563a74287b6755cc8839e41c0b63d4f24bc4R271.